_employment/employers/support-veteran-employees.md

```diff
@@ -3,6 +3,13 @@ title: Support Veteran Employees
 concurrence: incomplete
 template: 1-topic-landing
 ---
+<div class="action-bar">
+  <div class="row">
+    <div class="small-12 columns">
```

Comment:
Sure, but lets make that a bigger change across all the places we do this? It's not how it is currently elsewhere.